core/examples/driver/main.cpp

```diff
@@ -288,7 +240,7 @@ int main(int /*argc*/, char** /*argv*/)
       {
         std::cout << std::right << std::setw(2) << (*moduleIter)->GetModuleId() << std::left << " | ";
         std::cout << std::setw(20) << (*moduleIter)->GetName() << " | ";
-        std::cout << std::setw(9) << ((*moduleIter)->IsLoaded() ? "LOADED" : "UNLOADED");
+        std::cout << std::setw(9) << ((*moduleIter)->IsLoaded() ? "ACTIVE" : "RESOLVED");
```

Comment:
I suggest to keep it like this for now and agree with Jeff that we need to introduce a GetState() method later instead.